Library/Formula/rtorrent.rb

```diff
@@ -2,8 +2,11 @@
 
 class Rtorrent < Formula
   desc "Console-based BitTorrent client"
-  homepage "http://libtorrent.rakshasa.no/"
-  url "http://libtorrent.rakshasa.no/downloads/rtorrent-0.9.4.tar.gz"
+  # Both homepage and primary url have been down since at least ~April 2015
+  homepage "https://github.com/rakshasa/rtorrent"
+  url "https://github.com/rakshasa/rtorrent/archive/0.9.4.tar.gz"
+  mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/r/rtorrent/rtorrent_0.9.4.orig.tar.gz"
```

Comment:
The mirror and url aren't the same file so one or the other needs removed and the `sha1` may need updated (although change to `sha256` if you do that)